site/en/blog/no-spooky-cookies/index.md

```diff
@@ -0,0 +1,204 @@
+---
+title: No spooky cookies
+subhead: >
+  Cookies are best fresh, so what are the latest recipes to ensure you can
+  still enjoy spooky season without any stale cookies?
+description: >
+  Cookies are best fresh, so what are the latest recipes to ensure you can
+  still enjoy spooky season without any stale cookies?
+layout: 'layouts/blog-post.njk'
+date: 2022-10-24
+authors:
+  - rowan_m
+hero: 'image/VWw0b3pM7jdugTkwI6Y81n6f5Yc2/m2MdomFImr4xrDeBgRht.png'
+alt: >
+  A random assortment of spider webs, hourglasses and cookies - as time is
+  running out for third-party cookies on the web. Ho ho ho.
+tags:
+  - privacy
+---
+
+Cookies are always best when they are fresh, so what are the latest recipes to
+ensure you can still enjoy spooky season without any spooky, stale cookies?
+
+We are on the path towards phasing out third-party cookies across the web
+platform. That's a major milestone in tackling cross-site tracking, but it's
+part of a pretty long journey. Let's take a peek at how far we've come and what
+treats are in store in the future…
+
+On the surface, cookies provide a simple key-value store that's sent between the
+browser and the server. That can provide useful functionality on a site such as
+saving a preference: `theme=bats` or storing the session ID for a signed in
+user.
+
+{% Img src="image/VWw0b3pM7jdugTkwI6Y81n6f5Yc2/bwfhKOn5hODNKWWTFwH9.png", alt="A third-party cookie carrying a simple value like theme=bats or fav_pumpkins=us-nyc", width="800", height="450" %}
+
+If that cookie is being used on the same site that set it, then we tend to call
+that a first-party cookie. If it's being used as part of a different site than
+the one that set it, we call that a third-party cookie. For example, my
+`theme=bats` cookie would be first-party if I'm visiting that same site that set
+it, but if it's included in an iframe or other cross-site resource as part of a
+different site then it would be a third-party cookie.
+
+**Remember:** first-party or third-party is all about same-site or cross-site
```

Comment:
Would one of the aside components be okay for highlighting this?

Reply:
Will add!